SequenceAnalysis/build.gradle

```diff
@@ -12,6 +12,10 @@ repositories {
    maven {
       url "https://clojars.org/repo"
    }
+   // Added for org.clojars.chapmanb:sam dependency required by com.github.samtools:htsjdk
+   maven {
+      url "https://maven.scijava.org/content/groups/public"
+   }
```

Comment:
The jbrowse build.gradle has this same comment above its `clojars.org` repo declaration. Is `scijava.org` for a different dependency that we've been mirroring on artifactory?

Reply:
OK, I've removed scijava as it seems to not be needed here, but I won't be surprised if we find we need to reintroduce it as we merge this forward.